jsonrpc-parent/jsonrpc-documentation/src/test/java/io/yupiik/uship/jsonrpc/doc/openapi/OpenRPC2OpenAPITest.java

```diff
@@ -33,17 +33,208 @@ void convert() throws Exception {
             final var converter = new OpenRPC2OpenAPI(
                     Json.createBuilderFactory(Map.of()),
                     jsonb);
-            assertEquals("" +
-                            "{\"openapi\":\"3.0.3\",\"servers\":[{\"url\":\"http://localhost:8080/jsonrpc\",\"description\":\"JSON-RPC API\"}]," +
-                            "\"info\":{\"title\":\"Test\",\"description\":\"The test.\",\"version\":\"1\"}," +
-                            "\"paths\":{\"/test1\":{\"post\":{\"operationId\":\"test1\",\"summary\":\"\"," +
-                            "\"requestBody\":{\"content\":{\"application/json\":{\"schema\":{\"type\":\"object\"," +
-                            "\"properties\":{\"jsonrpc\":{\"type\":\"string\",\"default\":\"2.0\",\"description\":\"JSON-RPC version, should always be '2.0'.\"}," +
-                            "\"method\":{\"type\":\"string\",\"default\":\"test1\",\"description\":\"The JSON-RPC method name, should always be 'test1'\"}," +
-                            "\"params\":{\"type\":\"object\",\"properties\":{\"in\":{\"type\":\"string\"}}}},\"required\":[\"jsonrpc\",\"method\"]}}}}," +
-                            "\"responses\":{" +
-                            "\"200\":{\"description\":\"OK\",\"content\":{\"application/json\":{\"schema\":{\"type\":\"object\",\"properties\":{\"jsonrpc\":{\"type\":\"string\",\"default\":\"2.0\",\"description\":\"JSON-RPC version, should always be '2.0'.\"},\"result\":{\"type\":\"object\",\"properties\":{\"name\":{\"type\":\"string\"}}}},\"required\":[\"jsonrpc\",\"result\"]}}}}," +
-                            "\"501\":{\"description\":\"When something occurs. (Error code=1234)\",\"content\":{\"application/json\":{\"schema\":{\"type\":\"object\",\"properties\":{\"jsonrpc\":{\"type\":\"string\",\"default\":\"2.0\",\"description\":\"JSON-RPC version, should always be '2.0'.\"},\"result\":{\"type\":\"object\",\"properties\":{\"counter\":{\"type\":\"integer\"}}}},\"required\":[\"jsonrpc\",\"result\"]}}}}}}}},\"tags\":[],\"components\":{\"schemas\":{\"io_yupiik_uship_jsonrpc_core_servlet_JsonRpcServletTest_Foo\":{\"type\":\"object\",\"properties\":{\"name\":{\"type\":\"string\"}}}}}}",
+            assertEquals(""
```

Comment:
why is it formatted now? this shouldn't by default

Reply:
Only source are formatted, for readability. The expected still one line without space.

Reply:
yep but

1. it does not autoformat - whatever style you use
2. + moves from end of line to start of line

What about using jsonp to autoformat it in the test if you want? (`try (final var reader = Json.createreader(new stringreader(json))); final var writer = Json.createwriterfactory(map.of(JsonGenerator.PRETTY_FORMATTING, true)).createwriter(new StringWriter()))) { writer.write(reader.readOjbect()); }` then just use the string writer tostring result.

should be easy to put in a private method and reuse accross tests and solve this issue.